unit-test/sc_atsrq_tests.c

```diff
@@ -824,7 +824,7 @@ void SC_JumpAtsCmd_Test_SkipOneCmd(void)
     SC_Assert_ID_EQ(SC_OperData.AtsCtrlBlckAddr->CmdNumber,
                     SC_GetAtsCommandNumAtSeq(AtsIndex, SC_SEQUENCE_IDX_C(1))->CmdNum);
     SC_Assert_IDX_VALUE(SC_OperData.AtsCtrlBlckAddr->TimeIndexPtr, 1);
-    UtAssert_True(SC_AppData.NextCmdTime[0] == 0, "SC_AppData.NextCmdTime[0] == 0");
+    UtAssert_True(SC_AppData.NextCmdTime[SC_Process_ATP] == 0, "SC_AppData.NextCmdTime[SC_Process_ATP] == 0");
```

Comment:
Generic comment just FYI in the future - whenever you see UtAssert_True used in old tests replace it vs just update.  Goal is to use the unit test APIs that are self-documenting.  There's a unit test users guide that the cFS team should have w/ the details.  You should never have to specify the response string by hand.